src/main/java/dansplugins/factionsystem/eventhandlers/DamageHandler.java

```diff
@@ -60,32 +60,42 @@ public void handle(EntityDamageByEntityEvent event) {
      * 4) Players are not in the same faction but are not enemies.
      */
     private void handlePlayerVersusPlayer(Player attacker, Player victim, EntityDamageByEntityEvent event) {
+        Logger.getInstance().log("Handling damage between players.");
```

Comment:
i think we need add `debug: true` to config to prevent it spam when release

Reply:
Thanks for bringing this up! The method being called here actually only logs the message if the debug flag is enabled. I've made some changes to the Logger class to make this more clear.